README.md

```diff
@@ -60,7 +60,7 @@ Set up (the latest version of) [HAProxy](http://www.haproxy.org/) in Ubuntu syst
 * `haproxy_global_raw_options`: [default: `[]`]: Additional arbitrary lines to insert in the section
 
 * `haproxy_defaults_log`: [default: `global`]: Enable per-instance logging of events and traffic. `global` should be used when the instance's logging parameters are the same as the global ones. This is the most common usage
-* `haproxy_defaults_logformat`: [optional]: Allows you to customize the logs in http mode and tcp mode (e.g. `'"%{+Q}o\ %t\ %s\ %{-Q}r"'`)
+* `haproxy_defaults_logformat`: [optional]: Allows you to customize the logs in http mode and tcp mode (e.g. `%{+Q}o\ %t\ %s\ %{-Q}r`)
 * `haproxy_defaults_mode`: [default: `http`]: Set the running mode or protocol of the instance
```

Comment:
Is this branch not in sync?

Reply:
Strange, I will check it out

Reply:
@tersmitten Not sure what happened. I made a new merge request with the correct commits: https://github.com/Oefenweb/ansible-haproxy/pull/108